### PR TITLE
Adding standard Google Chrome support on MacOS

### PR DESCRIPTION
**Pull request by nmontesdeoca**
This is related to issue #689

### CODE REVIEW

**Comment by nmontesdeoca** [comment]
I signed it!

File: lighthouse-cli/chrome-finder.js [hunk 1]
```diff
@@ -34,15 +37,17 @@ module.exports = {
 
     execSync(
       `${LSREGISTER} -dump` +
-      ' | grep -i \'google chrome canary.app$\'' +
+      ' | grep -i \'google chrome[ canary]\{0,7\}.app$\'' +
```
Comment (brendankenny) [inline lighthouse-cli/chrome-finder.js]: hmm, I have to double escape like `\\{0,7\\}` for this to match anything on my machine.

Also, would it be better to just use a group here to match the exact substring? Something like `' | grep -i \'google chrome\\( canary\\)\\?.app$\'' +`

**Comment by brendankenny** [comment]
> With this change the standard Chrome installation will be favored, so I think you'll want to add something to the `priorities` `Map` to make sure Canary comes out on top.

hmm, this may take more planning than I thought, actually. I'm not totally sure of the reasoning behind the existing `priorities` map.

It currently favors a Chrome found under `/Applications/`, then under `~/Applications/`, then anywhere else before finally a Chrome under `/Volumes/`. It's not clear how that would interact with choosing Canary vs the default Chrome installation.

Should a Canary under `~/Applications/` be favored over a release Chrome in `/Applications/`? What about a Canary under `/Volumes/`?

@boopathi any thoughts?

File: lighthouse-cli/chrome-finder.js (hunk 1 above)
Comment (nmontesdeoca) [inline lighthouse-cli/chrome-finder.js]: This is much better for sure, I've applied the group to the regex. Using the option --select-chrome finds both canary and standard chrome in my mac.

**Comment by nmontesdeoca** [comment]
It's a little awful but allows to differentiate the priorities between standard and canary.

```javascript
const priorities = new Map([
  [/^\/Volumes\/.*[Cc]hrome [Cc]anary.app/, -1],
  [/^\/Volumes\/.*[Cc]hrome.app/, -2],
  [/^\/Applications\/.*[Cc]hrome [Cc]anary.app/, 101],
  [/^\/Applications\/.*[Cc]hrome.app/, 100],
  [new RegExp(`^${process.env.HOME}/Applications/.*[Cc]hrome [Cc]anary.app`), 51],
  [new RegExp(`^${process.env.HOME}/Applications/.*[Cc]hrome.app`), 50]
]);
```

**Comment by paulirish** [comment]
+1 on including chrome stable in the launcher. we can afford to do that now.

the priorities map above looks mostly good, favoring canary over stable and otherwise keeping the same sequence of volumes,/applications/,~/applications. 

AFAIK, you don't need to test both `[Cc]` and can rely on the case being consistent.

**Comment by nmontesdeoca** [comment]
Applied change inside priorities Map.

Per my understanding the priorities order is in this way:

1- Canary inside /Applications
2- Stable inside /Applications
3- Canary inside ~/Applications
4- Stable inside ~/Applications
5- Canary inside /Volumes
6- Stable inside /Volumes

Maybe ~/Applications should have more priority than /Applications ?

**Comment by nmontesdeoca** [comment]
Awesome!, Thank you.

**Pull request merged**